src/components/buttons/button-accessibility-image-alt.jsx

```diff
@@ -23,9 +23,55 @@ class ButtonAccessibilityImageAlt extends React.Component {
 	constructor(props) {
 		super(props);
 
-		const selection = props.context.editor
-			.get('nativeEditor')
-			.getSelection();
+		this.state = this._getInitialState();
+	}
+
+	/**
+	 * Lifecycle. Invoked once, only on the client, immediately after the initial rendering occurs.
+	 *
+	 * Focuses on the link input to immediately allow editing. This should only happen if the component
+	 * is rendered in exclusive mode to prevent aggressive focus stealing.
+	 *
+	 * @instance
+	 * @memberof ButtonAccessibilityImageAlt
+	 * @method componentDidMount
+	 */
+	componentDidMount() {
+		if (this.props.renderExclusive || this.props.manualSelection) {
+			// We need to wait for the next rendering cycle before focusing to avoid undesired
+			// scrolls on the page
+
+			this._focusAltInput();
+		}
+	}
+
+	/**
+	 * Lifecycle. Invoked when a component is receiving new props.
+	 * This method is not called for the initial render.
+	 *
+	 * @instance
+	 * @memberof ButtonAccessibilityImageAlt
+	 * @method componentWillReceiveProps
+	 */
+	componentWillReceiveProps() {
```

Comment:
Hey @georgel-pop-lr, why did you used `componentWillReceiveProps` in this case? The fix is working but I would know more about it :)

Reply:
Hi @diegonvs 

@georgel-pop-lr  is on PTO now.

The fix was created based on the ButtonLinkEdit: https://github.com/liferay/alloy-editor/blob/master/src/components/buttons/button-link-edit.jsx

More details here: https://issues.liferay.com/browse/PTR-2775?focusedCommentId=2576007&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-2576007

Could you plesae merge this change and release it in a new alloy-editor version?

Thanks, Roland

Reply:
Hi @diegonvs,

Yes as @rolandpakai  said I inspired myself on https://github.com/liferay/alloy-editor/blob/master/src/components/buttons/button-link-edit.jsx 

Let me know if anything else is needed.

Regards.

Reply:
Hi @diegonvs,

Will we upgrade alloyeditor on master as well? 

Also, let us know when a new version is available.

Regards.